src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -184,7 +184,7 @@ public void testPosition()
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
         verify(checkConfig, getPath("InputAbstractJavadocPosition.java"), expected);
         assertEquals("Invalid number of javadocs",
-            58, JavadocCatchCheck.javadocsNumber);
+            62, JavadocCatchCheck.javadocsNumber);
```

Comment:
Why did invalid javadocs number only increase by 4 when you added 10 invalid `nope` javadoc comments?

Reply:
You are right - the number should be 61, since I've added three new javadoc comments.
One `nope`, actually, is a javadoc comment.